### PR TITLE
Delete io.flutter.app android v1 embedding

### PR DESCRIPTION
**Pull request by GaryQian**
Based off of https://github.com/flutter/engine/pull/29248

go/flutter-delete-android-v1-embedding

Deletes the non-plugin related parts of io.flutter.app/* of the v1 android embedding.

This is the first and safest phase of the v1 embedding removal. There is already a build warning that warns against using v1 embedding which can be modified to remove the override option after this as apps will no longer compile without migration.

v1 plugin registry is left intact for now.

### CODE REVIEW

File: shell/platform/android/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -83,10 +83,10 @@
    * Interface for those objects that maintain and expose a reference to a {@code FlutterView} (such
    * as a full-screen Flutter activity).
    *
-   * <p>This indirection is provided to support applications that use an activity other than {@link
-   * io.flutter.app.FlutterActivity} (e.g. Android v4 support library's {@code FragmentActivity}).
-   * It allows Flutter plugins to deal in this interface and not require that the activity be a
-   * subclass of {@code FlutterActivity}.
+   * <p>This indirection is provided to support applications that use an activity other than
```
Comment (blasten) [inline shell/platform/android/io/flutter/view/FlutterView.java:86]: does this class also need to be deleted?

Comment (GaryQian) [reply shell/platform/android/io/flutter/view/FlutterView.java:86]: Yes. The intention is to remove this in multiple PRs in order of safety. Removing FlutterView also requires heavily modifying/removing plugin registry classes as they are dependent on this.

**Comment by skia-gold** [comment]
Gold has detected about 18 new digest(s) on patchset 4.
View them at https://flutter-engine-gold.skia.org/cl/github/32074

File: shell/platform/android/io/flutter/view/FlutterView.java (hunk 1 above)
Comment (blasten) [reply shell/platform/android/io/flutter/view/FlutterView.java:86]: oh ok

**Comment by skia-gold** [comment]
Gold has detected about 18 new digest(s) on patchset 5.
View them at https://flutter-engine-gold.skia.org/cl/github/32074

**Comment by blasten** [comment]
Please run this change with FROB

**Comment by GaryQian** [comment]
@jason-simmons I'm seeing typography golden changes, and I don't see how v1 removal can cause them. Do you know if this is related to SkParagraph and rollout of it?

**Review by blasten — APPROVED** [review]
LGTM

**Review by hamdikahloun — APPROVED** [review]
LGTM

**Pull request merged**